src/Form/FilePersisterServiceSettingsForm.php

```diff
@@ -56,6 +73,22 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#options' => $scheme_options,
       '#required' => TRUE
     ];
+    $form['object_file_path'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Relative Path for Persisting Digital Object Files'),
+      '#description' => $this->t('Path relative to the root of the storage scheme selected above where digital object files will be stored. Do not include beginning or ending slashes. Default is "@storage".
+                                  <br>Note that changing this setting will not affect the file storage locations for previously ingested objects. They will remain where they were.',
+        ['@storage' => StrawberryfieldEventInsertSubscriberDepositDO::DEFAULTOBJECTSTORAGEFILEPATH]),
+      '#default_value' => !empty($config_storage->get('object_file_path')) ? $config_storage->get('object_file_path') : StrawberryfieldEventInsertSubscriberDepositDO::DEFAULTOBJECTSTORAGEFILEPATH,
+      '#prefix' => '<span class="filepath-exec-path-validation"></span>',
+      '#ajax' => [
+        'callback' => [$this, 'validateObjectFilePath'],
+        'effect' => 'fade',
+        'wrapper' => 'objectfilepath-exec-path-validation',
```

Comment:
The `exec` is a bit confusing. Better remove since there is no execution at all

Reply:
Yep, a lazy copy-paste mistake.

Reply:
Fixed.